databuilder/models/table_metadata.py

```diff
@@ -67,14 +95,16 @@ def __init__(self,
         self.description = description
         self.type = col_type
         self.sort_order = sort_order
-        self.tags = tags
+        self.tags = TagMetadata.parse_tags_from_strings(tags)
 
     def __repr__(self):
         # type: () -> str
-        return 'ColumnMetadata({!r}, {!r}, {!r}, {!r})'.format(self.name,
-                                                               self.description,
-                                                               self.type,
-                                                               self.sort_order)
+        return 'ColumnMetadata({!r}, {!r}, {!r}, {!r}, {!r})'.format(self.name,
+                                                                     self.description,
+                                                                     self.type,
+                                                                     self.sort_order,
+                                                                     [tag for tag in self.tags]
```

Comment:
let's hold off the badge for column for now as the pr/issue is mainly for adding quick start for badge. And we don't have a good e2e story around column tag/badge.

Reply:
ok sounds good